wp-pgp-encrypted-emails.php

```diff
@@ -913,20 +1248,20 @@ private static function shouldSign ($to) {
      */
     private static function prepareMail ($to, $subject, $message, $headers, $attachments) {
         $pub_key       = false;
+        $pub_cert      = false;
         $erase_subject = false;
 
+        // Note: If admin email is also a user and key only exists with user, this won't find it.
```

Comment:
This comment worries me. Are you saying here that there could be a situation in which the plugin will behave unexpectedly if the site's admin email address matches a user's email address?